source/docs/getting-started/getting-started-frc-control-system/control-system-hardware.rst

```diff
@@ -10,7 +10,9 @@ The goal of this document is to provide a brief overview of the hardware compone
 Overview of Control System
 -------------------------------
 
-.. image:: images/control-system-hardware/FRC-Control-System-Layout.svg
+.. figure:: images/control-system-hardware/FRC-Control-System-Layout.svg
+
+  Diagram countacy of FRC Team 3161 and Stefen Acepcion.
```

Comment:
Indentation looks wrong here.

Reply:
It looks like it builds properly, what do you mean?

Reply:
Indentation should be 3 spaces not 2. It should align with the first character of ``figure``

Reply:
Huh, I added 3 spaces to the indent in 2db721e, but the build failed with the error:
`/home/vsts/work/1/s/source/docs/getting-started/getting-started-frc-control-system/how-to-wire-a-robot.rst:13: WARNING: Figure caption must be a paragraph or empty comment.
`
- Interesting that there are also 3 spaces in control-system-hardware, line 15, but no error was thrown.
- Builds passed in 09c9a8b and locally with 2 spaces for indents, so the third space seems to be the cause of the build error.